server/authz/policy.rego

```diff
@@ -643,6 +643,36 @@ allow {
   action == read
 }
 
```

Comment:
Not sure why these changes are showing on this PR. No need to review this file again. They are the same as what has previously [merged](https://github.com/fleetdm/fleet/pull/18513).